src/Command/ConfigureCommand.php

```diff
@@ -25,8 +25,13 @@
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
+ */
 final class ConfigureCommand extends BaseCommand
 {
+    const NONINTERACTIVE_MODE_ERROR = 'Infection config generator requires an interactive mode.';
+
```

Comment:
[Used in the test](https://github.com/infection/infection/pull/285/files#diff-c50be878dbd21bc70fc3c4da7deedccfR80) to confirm output of failing configure command.